dnsmenace.py

```
@@ -448,70 +665,103 @@ def compare(
+    by querying the same domain from servers in different countries and
+    highlighting any discrepancies.
 
     Example:
         dnsmenace compare google.com -c US -c CN -c RU -c IR
-        dnsmenace compare example.com --countries US DE JP
+        dnsmenace compare example.com --countries US DE JP --doh
     """
     print_banner()
 
-    console.print(f"\n[bold]Comparing DNS responses for:[/bold] {domain}")
+    console.print(f"\n[bold cyan]Geo-Censorship Detection Mode[/bold cyan]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)